robotframework_reportportal/service.py

```diff
@@ -27,207 +30,134 @@ class RobotService(object):
         "DEBUG": "DEBUG"
     }
 
-    stack = []
-    logs = []
-
     @staticmethod
-    def init_service(endpoint, project, uuid):
+    def init_service(endpoint, project, uuid, log_batch_size):
         if RobotService.rp is None:
             logging.debug(
-                msg="ReportPortal - Init service: "
-                    "endpoint={0}, project={1}, uuid={2}".format(endpoint,
-                                                                 project,
-                                                                 uuid))
-            RobotService.rp = ReportPortalService(endpoint=endpoint,
-                                                  project=project,
-                                                  token=uuid)
+                "ReportPortal - Init service: "
```

Comment:
Please, walk through all debug messages and use one log style: `canonical` like bellow or using `str.format`.